### ISSUE TITLE
Webhooks feature

### ISSUE THREAD

**Issue by headmin** [brysontyrrell/PatchServer#4]
Having an option to send out a Webhook with context would be a great feature to see in PatchServer.
Our use specific use-case would be feed the "patch info + context" as event into our centralized event stream processing and monitoring workflows. 
But in general Webhooks would enable many other use cases as well for the community.

**Comment by brysontyrrell** [comment]
@headmin - Would the following JSON payload fit the request?

```
{
    "event": "<event-type>",
    "timestamp": 123456789,
    "software_title": "<title-name>",
    "version:": "<version>",
    "url": "http://<server>/jamf/v1/patch/<id>"
}
```

**event-type** would be either **new_software_title** or **new_version**.

**Comment by brysontyrrell** [comment]
Webhooks have been implemented with the merge #7 (v0.7.0).
